TSC-Charter.md

```diff
@@ -162,12 +181,13 @@ for dissent from the consensus.
 If an agenda item cannot reach a consensus a TSC member can call for
 either a closing vote or a vote to table the issue to the next meeting.
 The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue. Simple majority wins, with the following
-exceptions, which will require the affirmative vote of two-thirds of the
-members of the TSC to pass:
+discussion will continue. Simple majority wins with exception given only to
+removal of a member from the TSC, which shall require a two-thirds majority
+vote to approve.
```

Comment:
I think it's weird and hard to defend that you can get in on a hypothetical 51% majority but expulsion requires 66%.  It should either stay the way it is or it should be 51% for both.